helm/charts/infra/templates/NOTES.txt

```diff
@@ -37,6 +37,14 @@
 
 {{- end }}
 
+{{- if and .Values.server.persistence.enabled (not .Values.server.config.dbEncryptionKey) .Release.IsUpgrade }}
```

Comment:
Adding a hint here is a great idea. I'd even recommend us linking to upgrade docs for _any_ install so users can look that up for manual migration instructions.

Reply:
I was thinking that too. Will make that change

Reply:
On second thought, it might not make the most sense since NOTES.txt is only outputted after an upgrade. This is more of a troubleshooting hint in case something goes wrong

Reply:
Sounds good!